### ISSUE TITLE
Simple_GP_Regression

### ISSUE THREAD

**Issue by ssaxenaANL** [cornellius-gp/gpytorch#1662]
I am trying to run simple_GP_Regression.ipynb. The kernel is always breaking at training step: 
<img width="451" alt="GP" src="https://user-images.githubusercontent.com/61334206/122581078-4c23ac00-d01c-11eb-8f3b-fce5ea83f21e.PNG">

The loss calculation step is keep failing:
loss = -mll(output, train_y)

Can you please help? What might cause this issue?

**Comment by gpleiss** [comment]
This is a duplicate of #1663
